src/server.c

```diff
@@ -98,9 +98,26 @@ void nolocks_localtime(struct tm *tmp, time_t t, time_t tz, int dst);
 
 /* Low level logging. To use only for very big messages, otherwise
  * serverLog() is to prefer. */
+#define ANSI_BLACK "\033[30m"
```

Comment:
i guess that we're set on the colors we wanna use, and we should delete the ones we don't.